docker-compose-tests.yml

```diff
@@ -15,17 +15,16 @@ services:
     links:
       - gp-finder-db:es
 
-  travis-tests:
+  ci-tests:
     extends:
       file: docker-compose.yml
       service: base-app
-    command: sh ./scripts/container-travis-tests
+    command: scripts/container/ci-tests
     volumes:
-      - gp-finder-travis-tests-node_modules:/code/node_modules
+      - gp-finder-ci-tests-node_modules:/code/node_modules
```

Comment:
If the service is renamed to `test-ci` I'd suggest renaming this to match i.e. `gp-finder-test-ci-node_modules` (not that it really matters...).